<?xml version="1.0"?>
<launch>

    <arg name="input_points_topic" default="/velodyne_packets" />
    <arg name="min_range" default="1.0" />
    <arg name="odom_link" default="/world" />
    <arg name="odom_child_link" default="/state_base_link" />

    <!-- Run monte carlo localization -->
    <!--include file="$(find ndt_calibration)/launch/visualize.launch" /-->
    <node name="velodyne_converter" pkg="graph_map" type="velodyne_converter" output="screen">

        <!-- select one of the following types-->
        <param name="disable_compensation" value="false" />
        <param name="use_pointtype_xyzir" value="false" />

        <param name="laser_tf" value="/velodyne" />

        <!-- Input topic name-->
        <param name="input_points_topic" value="$(arg input_points_topic)" />
        <!-- Output topic name, this is passed to the mapping node -->
        <param name="output_points_topic" value="/velodyne_points" />
        <param name="output_points_link" value="/velodyne" />
        <param name="calibration" value="$(find graph_map)/config/velo32.yaml" />
        <param name="sensor_offset_topic" value="/set_sensor_offset" />

        <param name="odom_link" value="$(arg odom_link)" />
        <param name="odom_child_link" value="$(arg odom_child_link)" />

        <param name="min_range" value="$(arg min_range)" />
        <param name="max_range" value="130.0" />

        <!-- pass path to graph_map (.map)-->
        <param name="sensor_pose_x" value="1.179206" />
        <param name="sensor_pose_y" value="-0.28589" />
        <param name="sensor_pose_z" value="2.0" />
        <param name="sensor_pose_r" value="0" />
        <param name="sensor_pose_p" value="0" />
        <param name="sensor_pose_t" value="-1.58804" />

    </node>


</launch>